### ISSUE TITLE
Colbalt and Ardite chickens not dropping ingots.

### ISSUE THREAD

**Issue by aDubiousNotion** [GenDeathrow/MoreChickens#14]
EDIT 2: After some testing, I think I've figured out the issue.

If I take an adult chicken that has not yet dropped its item for the first time and place it in a nesting pen, it won't lay that item.  It will lay manure, feathers, and eggs, but not the drop item.
If, on the other hand, I let it roam free until it makes its first item drop and then place it in a nesting pen, it will drop its item.

To test this I bred two copies of the same type of chicken and let them both grow to adults.  I immediately placed one in a nesting pen and left the other roaming.  The one in the pen never laid its item.  After the one roaming laid the item, I placed it in a pen and it continued to lay the item.  I tried this with Cobalt, Ardite, Manyullyn, Pig Iron, and Knight Slime, and it held true for all of them.

------

EDIT: Some of the Cobalt and Ardite chickens have begun dropping ingots.  Over a period of about 6 hours yesterday they were not dropping, but this afternoon a few of the Cobalt and one of the Ardite began laying ingots.  Others Cobalt and Ardite continue not to, however.

-------

I bred some Cobalt and Ardite chickens and put them in nesting pens.  They lay eggs, manure, and feathers, but they don't lay ingots.  Here are their config file settings:

```
arditechicken {
    # Item amount to be laid/dropped. [range: 1 ~ 64, default: 1]
    I:dropItemAmount=1

    # Item amount to be laid/dropped. [range: -2147483648 ~ 2147483647, default: 1]
    I:dropItemMeta=1

    # Item name to be laid/dropped. [default: tconstruct:ingots]
    S:dropItemName=tconstruct:ingots

    # Item amount to be laid/dropped. [range: 1 ~ 64, default: 1]
    I:eggItemAmount=1

    # Item amount to be laid/dropped. [range: -2147483648 ~ 2147483647, default: 1]
    I:eggItemMeta=1

    # Item name to be laid/dropped. [default: tconstruct:ingots]
    S:eggItemName=tconstruct:ingots

    # Is chicken enabled? [default: true]
    B:enabled=true

    # Scale time to lay an egg. [range: 0.01 ~ 100.0, default: 1.0]
    S:layCoefficient=1.0

    # First parent, empty if it's base chicken. [default: BlazeChicken]
    S:parent1=BlazeChicken

    # First parent, empty if it's base chicken. [default: MagmaChicken]
    S:parent2=MagmaChicken

    # Chicken spawn type, can be: NORMAL,SNOW,NONE,HELL [default: NONE]
    S:spawnType=NONE
}
```

```
colbaltchicken {
    # Item amount to be laid/dropped. [range: 1 ~ 64, default: 1]
    I:dropItemAmount=1

    # Item amount to be laid/dropped. [range: -2147483648 ~ 2147483647, default: 0]
    I:dropItemMeta=0

    # Item name to be laid/dropped. [default: tconstruct:ingots]
    S:dropItemName=tconstruct:ingots

    # Item amount to be laid/dropped. [range: 1 ~ 64, default: 1]
    I:eggItemAmount=1

    # Item amount to be laid/dropped. [range: -2147483648 ~ 2147483647, default: 0]
    I:eggItemMeta=0

    # Item name to be laid/dropped. [default: tconstruct:ingots]
    S:eggItemName=tconstruct:ingots

    # Is chicken enabled? [default: true]
    B:enabled=true

    # Scale time to lay an egg. [range: 0.01 ~ 100.0, default: 1.0]
    S:layCoefficient=1.0

    # First parent, empty if it's base chicken. [default: NetherwartChicken]
    S:parent1=NetherwartChicken

    # First parent, empty if it's base chicken. [default: GhastChicken]
    S:parent2=GhastChicken

    # Chicken spawn type, can be: NORMAL,SNOW,NONE,HELL [default: NONE]
    S:spawnType=NONE
}
```

**Comment by GenDeathrow** [comment]
I have never seen or heard of this issue. also it shouldn't only affect one type of chicken. it should happen to each and every chicken in pens. All the pens do is collect drops from the chickens. so the chickens drops behave the same even if they are in the pens or not.

**Comment by aDubiousNotion** [comment]
Ah, sorry, my edit was meant to say that it's not just Cobalt and Ardite.  I didn't notice the issue until Cobalt and Ardite because of the long initial egg time on those chickens.

For the early chickens they would naturally lay their drop before I had a pen ready, so I never encountered the issue.  Cobalt and Ardite on the other hand take ~70 min for the first drop so I got them in pens before that and found the problem.  I went back and bred some earlier chickens to check and found that it happens for them as well.

To reproduce:
1) Hatch two chickens of the same type.  Wait until they grow to adults.
2) Immediately place one in a nesting pen before the initial drop timer runs out.
3) Leave the other chicken wandering.
4) Wait until the wandering chicken makes its first drop.
5) Place the wandering chicken into a nesting pen.

The chicken placed immediately into the pen will lay manure, feathers, and eggs, but not its item.
The chicken allowed to lay once naturally will lay its item in the pen.

Here is another player who encountered the same issue: https://www.reddit.com/r/feedthebeast/comments/64mo9k/sf3_chickens_help/

That player was playing Sky Factory 3, which is also the pack I'm playing, so there's that as a common factor.
